src/components/Exchange/ExchangeTVChart.js

```diff
@@ -117,13 +118,21 @@ export default function ExchangeTVChart(props) {
       .filter((p) => p.indexToken.address === chartToken.address)
       .map((position) => {
         const longOrShortText = position.isLong ? t`Long` : t`Short`;
+        const liquidationPrice = getLiquidation({
```

Comment:
mm, why the function was renamed to getLiquidation?) For me it looks less clear that this is about liq.price

Reply:
It was just for testing purpose, should have removed the older function and changed it to getLiquidationPrice. Will do that now.